docs/projects/amt/reporting-standard/latest.md

```diff
@@ -97,15 +97,36 @@ There can be multiple instruments. For each instrument the following fields are
        set.
 
 7. `description` (OPTIONAL, string). A short description of the system.
-8. `labels` (OPTIONAL, list). This fields allows to store meta information about a system. There can be multiple labels.
+8. `ai_act_profile` (OPTIONAL). Information about the system in relation to the EU AI Act.
+    The contents of this field can be retrieved by traversing the AI Act decision tree or can be specified by the user.
+
+    1. `type` (OPTIONAL, enum[string]). The type of the system should be chosen from:
+    `["AI-systeem", "AI-systeem voor algemene doeleinden", "AI-model voor algemene doeleinden", "algoritme", "geen algoritme"]`.
+    2. `open_source` (OPTIONAL, enum[string]). Whether the system is open source or not.
+    Options are `["open-source", "geen open-source"]`.
+    3. `publication_category` (OPTIONAL, enum[string]). The publication category of the system should be chosen from:
+    `["impactvol algoritme", "niet-impactvol algoritme",  "hoog-risico AI",
+    "geen hoog-risico AI", "verboden AI", "uitzonderingsgrond: AI-verordening niet van toepassing"]`.
```

Comment:
Should "uitzonderingsgrond: AI-verordening niet van toepassing" also be removed? I see it is also still present in the decision-tree repo decision-tree.yaml (line 124).

Reply:
We decided to remove this label from both repositories.